llvm-spirv/lib/SPIRV/CMakeLists.txt

```diff
@@ -1,54 +1,107 @@
-add_llvm_library(LLVMSPIRVLib
-  LLVMSPIRVOpts.cpp
-  LLVMToSPIRVDbgTran.cpp
-  Mangler/FunctionDescriptor.cpp
-  Mangler/Mangler.cpp
-  Mangler/ManglingUtils.cpp
-  Mangler/ParameterType.cpp
-  OCLToSPIRV.cpp
-  OCLTypeToSPIRV.cpp
-  OCLUtil.cpp
-  VectorComputeUtil.cpp
-  SPIRVLowerBitCastToNonStandardType.cpp
-  SPIRVLowerBool.cpp
-  SPIRVLowerConstExpr.cpp
-  SPIRVLowerMemmove.cpp
-  SPIRVLowerOCLBlocks.cpp
-  SPIRVLowerSaddWithOverflow.cpp
-  SPIRVReader.cpp
-  SPIRVRegularizeLLVM.cpp
-  SPIRVToLLVMDbgTran.cpp
-  SPIRVToOCL.cpp
-  SPIRVToOCL12.cpp
-  SPIRVToOCL20.cpp
-  SPIRVUtil.cpp
-  SPIRVWriter.cpp
-  SPIRVWriterPass.cpp
-  PreprocessMetadata.cpp
-  libSPIRV/SPIRVBasicBlock.cpp
-  libSPIRV/SPIRVDebug.cpp
-  libSPIRV/SPIRVDecorate.cpp
-  libSPIRV/SPIRVEntry.cpp
-  libSPIRV/SPIRVFunction.cpp
-  libSPIRV/SPIRVInstruction.cpp
-  libSPIRV/SPIRVModule.cpp
-  libSPIRV/SPIRVStream.cpp
-  libSPIRV/SPIRVType.cpp
-  libSPIRV/SPIRVValue.cpp
-  LINK_COMPONENTS
-    Analysis
-    BitWriter
-    CodeGen
-    Core
-    Demangle
-    IRReader
-    Linker
-    Passes
-    Support
-    TransformUtils
+if(LLVM_LINK_LLVM_DYLIB)
+  add_llvm_library(LLVMSPIRVLib STATIC DISABLE_LLVM_LINK_LLVM_DYLIB
+    LLVMSPIRVOpts.cpp
```

Comment:
This list of 36 files is copied twice.
Please create a variable for them, then reuse it in 2 places like it is done here: https://github.com/intel/llvm/blob/sycl/sycl/source/CMakeLists.txt#L116